scavenger-entity/src/main/kotlin/com/navercorp/scavenger/entity/ExportSnapshotMethodEntity.kt

```diff
@@ -0,0 +1,37 @@
+package com.navercorp.scavenger.entity
+
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Table
+
+@Table("snapshots")
+data class ExportSnapshotMethodEntity(
+    @Column("filterInvokedAtMillis")
+    val filterInvokedAtMillis: Long,
+
+    @Column("packages")
+    val packages: String? = null,
+
+    @Column("status")
+    val status: String? = null,
+
+    @Column("excludeAbstract")
+    val excludeAbstract: String? = null,
+
+    @Column("parent")
+    val parent: String? = null,
```

Comment:
It is a not null field.

Reply:
I fixed this and other type/nullable issues on commit 7d70bf1.